--- /usr/src/distrib/sets/lists/xbase/md.amd64	2018-07-22 19:47:26.670917387 +0900
+++ md.amd64	2018-09-21 11:35:35.143406419 +0900
@@ -1,23 +1,23 @@
 # $NetBSD: md.amd64,v 1.16 2016/08/17 03:10:22 mrg Exp $
-./usr/X11R7/lib/libI810XvMC.so			-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libI810XvMC.so.1		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libI810XvMC.so.1.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libIntelXvMC.so			-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libIntelXvMC.so.1		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libIntelXvMC.so.1.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMC.so		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMC.so.1		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMC.so.1.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMCPro.so		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMCPro.so.1		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libchromeXvMCPro.so.1.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_amdgpu.so		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_amdgpu.so.1		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_amdgpu.so.1.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_intel.so			-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_intel.so.0		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_intel.so.0.2		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_nouveau.so		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_nouveau.so.3		-unknown-		xorg,compatx11file
-./usr/X11R7/lib/libdrm_nouveau.so.3.0		-unknown-		xorg,compatx11file
-./usr/X11R7/share/pciids/pci.ids		-unknown-		obsolete
+./usr/X11R7/lib/libI810XvMC.so			xbase-lib-i810		xorg,compatx11file
+./usr/X11R7/lib/libI810XvMC.so.1		xbase-lib-i810		xorg,compatx11file
+./usr/X11R7/lib/libI810XvMC.so.1.0		xbase-lib-i810		xorg,compatx11file
+./usr/X11R7/lib/libIntelXvMC.so			xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libIntelXvMC.so.1		xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libIntelXvMC.so.1.0		xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMC.so		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMC.so.1		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMC.so.1.0		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMCPro.so		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMCPro.so.1		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libchromeXvMCPro.so.1.0		xbase-lib-chrome		xorg,compatx11file
+./usr/X11R7/lib/libdrm_amdgpu.so		xbase-lib-amdgpu		xorg,compatx11file
+./usr/X11R7/lib/libdrm_amdgpu.so.1		xbase-lib-amdgpu		xorg,compatx11file
+./usr/X11R7/lib/libdrm_amdgpu.so.1.0		xbase-lib-amdgpu		xorg,compatx11file
+./usr/X11R7/lib/libdrm_intel.so			xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libdrm_intel.so.0		xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libdrm_intel.so.0.2		xbase-lib-intel		xorg,compatx11file
+./usr/X11R7/lib/libdrm_nouveau.so		xbase-lib-nouveau		xorg,compatx11file
+./usr/X11R7/lib/libdrm_nouveau.so.3		xbase-lib-nouveau		xorg,compatx11file
+./usr/X11R7/lib/libdrm_nouveau.so.3.0		xbase-lib-nouveau		xorg,compatx11file
+./usr/X11R7/share/pciids/pci.ids		xbase-obsolete		obsolete
